docrepo.bib

```diff
@@ -64,23 +86,8 @@ @MISC{2019ivoa.spec.0927D
       adsnote = {Provided by the SAO/NASA Astrophysics Data System}
 }
 
-@MISC{2019ivoa.rept.0520D,
```

Comment:
Was this delete intentional? The document is still in ADS with the same bibcode: https://ui.adsabs.harvard.edu/abs/2019ivoa.rept.0520D/abstract